.gitignore

```diff
@@ -31,3 +31,4 @@ proguard/
 MPChartLib
 
 fw.dirs
+.project
```

Comment:
If you intend to keep the .project, .classpath and .settings files/folders only locally, you need to add all of them to .gitignore and remove the already committed files and folders.

Reply:
Hello, added all auto-generated stuff in .gitignore and removed them from git too.